tests/node_converters/lrn_test.py

```diff
@@ -0,0 +1,36 @@
+from random import randrange
+import numpy as np
+import onnx
+
+from tests.utils.common import check_onnx_model  # pylint: disable=wrong-import-position
```

Comment:
You don't need to use a `pylint: disable` here.

Reply:
It does seem to be fine now--previously when I was writing this, it started giving me wrong import position warnings, so I threw that in there to silence them. However, when I run linting now, that error isn't popping up. Huh! I'll remove the disable comments.